spec/index.adoc

```diff
@@ -2872,13 +2894,13 @@ State after::
 ....
 if
```

Comment:
Is this if-condition aligned with what the mainnet implementation does? @bogwar

Reply:
Changing `New_balance > if Is_response then 0 else freezing_limit(S, CM.callee)` to `New_balance >= freezing_limit(S, CM.callee)` looks sufficient to me for now.